backend/main.py

```
@@ -68,6 +68,86 @@ def create_access_token(data: dict, expires_delta: Optional[timedelta] = None):
     return encoded_jwt
 
 
+def _upload_image_from_base64(image_base64: str, folder: str = "spots") -> str:
+    """
+    base64文字列から画像をアップロードし、公開URLを返す。
+    バリデーションエラーはHTTPExceptionで返却する。
+    """
+    # base64サイズ制限（約10MB相当）
```

Ruff (0.14.8)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)